extensions/markdown-language-features/src/preview/documentRenderer.ts

```diff
@@ -103,7 +105,7 @@ export class MdDocumentRenderer {
 				${this._getStyles(resourceProvider, sourceUri, config, imageInfo)}
 				<base href="${resourceProvider.asWebviewUri(markdownDocument.uri)}">
 			</head>
-			<body class="vscode-body ${config.scrollBeyondLastLine ? 'scrollBeyondLastLine' : ''} ${config.wordWrap ? 'wordWrap' : ''} ${config.markEditorSelection ? 'showEditorSelection' : ''}">
+			<body class="vscode-body ${config.scrollBeyondLastLine ? 'scrollBeyondLastLine' : ''} ${config.wordWrap ? 'wordWrap' : ''} ${config.markEditorSelection ? 'showEditorSelection' : ''} ${isWeb ? 'style="overscroll-behavior-x: none;"' : ''}">
```

Comment:
Can we just disable `overscroll-behavior-x`. I don't think we want it on desktop either

Reply:
sure~